.ci/packaging.groovy

```diff
@@ -283,9 +281,7 @@ def pushCIDockerImages(Map args = [:]) {
     } else if (env?.BEATS_FOLDER?.endsWith('osquerybeat')) {
       tagAndPush(beatName: 'osquerybeat', arch: arch, variants: defaultVariants)
     } else if ("${env.BEATS_FOLDER}" == "packetbeat"){
-      tagAndPush(beatName: 'packetbeat', arch: arch, variants: defaultVariants)
-    } else if ("${env.BEATS_FOLDER}" == "x-pack/elastic-agent") {
-      tagAndPush(beatName: 'elastic-agent', arch: arch, variants: defaultVariants + completeVariant + cloudVariant)
+      tagAndPush(beatName: 'packetbeat', arch: arch)
```

Comment:
Can you also remove the lines https://github.com/elastic/beats/pull/30559/files#diff-5edb5ab93fc95960129eecb2ee3d8763a9c5e02b56e5b80126e0c5a52296ee8fR270-R272 

Those lines contain the definition for completeVariant + cloudVariant which they are elastic-agent specific